src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html

```diff
@@ -905,10 +905,7 @@ <h5 class="col-sm-2">
 
 
     
-        <div id="statusMessage" class="alert alert-warning">
-            The feedback session has been updated.
-        </div>
-        <script type="text/javascript" src="/js/statusMessage.js"></script>
+        <div id="statusMessage" style="display: none;"></div>
```

Comment:
Well, previously the test was redirect result. Now it became ajax result so the content in `textarea` is not updated. So I refresh the page and the statusMessage disappear as a result. You can check the `AdminHomePageUiTest.java`.